recipe/meta.yaml

```diff
@@ -81,7 +81,8 @@ outputs:
         - dataclasses  # [py<37]
         - filelock
         - frozenlist
-        - grpcio >=1.28.1, <=1.43.0
+        - grpcio >=1.28.1, <=1.43.0  # [py<310]
+        - grpcio >=1.42.0, <=1.43.0  # [py>=310]
```

Comment:
I see that this comes from upstream `setup.py`, but it's _very_ likely unnecessary. There simply aren't builds that old for python 3.10, and so the old condition should be more than enough. Based on looking for "py310" in [here](https://github.com/regro/libcfgraph/tree/master/artifacts/grpcio/conda-forge/linux-64), the first 3.10 builds were for 1.41.1. I presume the upstream condition just reflects that first availability.

Reply:
So to be clear, your suggestion is to use `grpcio >=1.28.1, <=1.43.0` for all pythons?

Reply:
If so I think that should be fine.

Reply:
> So to be clear, your suggestion is to use `grpcio >=1.28.1, <=1.43.0` for all pythons?

It would have exactly the same effect, yes. :)

Reply:
Ok - then let's override the upper bound and add a comment

Reply:
Updated, PTAL